Makefile

```diff
@@ -55,7 +55,7 @@ cleanbisect:
 	rm -Rf _coverage
 	find ./ -name 'bisect*.out' -delete
 
-SRCS=$(shell \ls {src/{,import/},tools/ocamlformat-diff/}*.ml{,i})
+SRCS=$(shell \ls {src/{,import/,config/,compat/},tools/ocamlformat-diff/}*.ml{,i})
```

Comment:
This gives 
```
ls: cannot access 'src/config/*.mli': No such file or directory
ls: cannot access 'src/compat/*.mli': No such file or directory
```
I suggest you change to
```
SRCS=$(shell find src tools/ocamlformat-diff -name '*.ml' -or -name '*.mli')
```

Reply:
Also, `tools/gen_version.ml` could be included.

Reply:
We do not include tools/gen_version.ml on purpose (see https://github.com/ocaml-ppx/ocamlformat/pull/732)